Tiltfile

```diff
@@ -108,6 +108,20 @@ docker_build('spoonerisms', 'spoonerisms',
   ]
 )
 
+# These two services run on the same container -- we can live update them both!
+docker_build('doggos', 'doggos',
+  live_update=[
+    sync('doggos', '/go/src/github.com/windmilleng/servantes/doggos'),
+    run('go install github.com/windmilleng/servantes/doggos'),
+    restart_container(),
+])
+docker_build('sidecar', 'sidecar',
+  live_update=[
+    sync('sidecar/src/', '/src/'),
+    run('cargo build --release'),
```

Comment:
Doesn't really matter but I bet IRL you'd just do a `cargo build` here since it would be faster and keep debug symbols.

Reply:
ha idk how to rust. I'll do that!

Reply:
...wait nvm, it doesn't update properly if i do that 🤔

Reply:
curious why the thing you said doesn't work, but don't particularly feel like digging into it now, so imma merge the thing that works.